lib/ContextModule.js

```diff
@@ -435,6 +435,7 @@ class ContextModule extends Module {
 			defaultObject: "redirect-warn"
 		};
 		this.buildInfo = {
+			strict: true,
```

Comment:
We should pass the option here or infer them from something, otherwise it is a breaking change

Reply:
I think most of the code in `ContextModule` is generated by Webpack. So it's fine to make it from non-strict to strict.